content/docs/user-guide/experiment-management/cleaning-experiments.md

```diff
@@ -147,6 +165,9 @@ $ dvc exp show --all-tags
 
 ```dvc
 $ dvc exp gc --all-tags
+WARNING: This will remove all experiments except those derived from the workspace of the current repo. Run queued experiments will be removed.
```

Comment:
```suggestion
WARNING: This will remove all experiments except those derived from any tags of the current repo. Run queued experiments will be removed.
```